tests/fixtures/disallow-multiple-var-decl/good/for.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable ember-suave/prefer-destructuring */
```

Comment:
Yeah, I'm not sure why it doesn't like that, but it gives an error without those disable comments